ganttproject-tester/test/biz/ganttproject/impex/csv/CsvColorExImportTest.java

```diff
@@ -0,0 +1,112 @@
+package biz.ganttproject.impex.csv;
+
+import biz.ganttproject.core.model.task.TaskDefaultColumn;
+import biz.ganttproject.core.option.BooleanOption;
+import com.google.common.base.Supplier;
+import junit.framework.TestCase;
+import net.sourceforge.ganttproject.GanttTask;
+import net.sourceforge.ganttproject.TestSetupHelper;
+import net.sourceforge.ganttproject.io.CSVOptions;
+import net.sourceforge.ganttproject.language.GanttLanguage;
+import net.sourceforge.ganttproject.resource.HumanResourceManager;
+import net.sourceforge.ganttproject.roles.RoleManagerImpl;
+import net.sourceforge.ganttproject.task.CustomColumnsManager;
+import net.sourceforge.ganttproject.task.TaskManager;
+
+import java.awt.*;
+import java.io.*;
+import java.nio.file.FileAlreadyExistsException;
+import java.text.SimpleDateFormat;
+import java.util.Map;
+
+public class CsvColorExImportTest extends TestCase {
```

Comment:
I suggest adding portions of this unit test into `GPCsvExportTest` and `GPCsvImportTest`. We have the necessary test harness there, inclusing locale apis and so on, so it must be easy and will let us keep uniform way of testing CSV stuff.